Tools/macOS/Info.plist.in

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+
+		<key>CFBundleName</key>
+		<string>JASP</string>
+
+		<key>CFBundleInfoDictionaryVersion</key>
+		<string>6.0</string>
+
+		<key>CFBundleDevelopmentRegion</key>
+		<string>English</string>
+
+		<key>CFBundleVersion</key>
+		<string>@JASP_VERSION@</string>
+
+		<key>CFBundleShortVersionString</key>
+		<string>@JASP_SHORT_VERSION@</string>
+		
+		<key>NSPrincipalClass</key>
+		<string>NSApplication</string>
+
+		<key>CFBundleIconFile</key>
+		<string>icon.icns</string>
+
+		<key>CFBundlePackageType</key>
+		<string>APPL</string>
+
+		<key>CFBundleGetInfoString</key>
+		<string>Created by Qt/CMake</string>
+
+		<key>CFBundleExecutable</key>
+		<string>JASP</string>
+
+		<key>CFBundleIdentifier</key>
+		<string>org.jasp-stats.jasp</string>
+
+		<key>LSMinimumSystemVersion</key>
+		<string>@CMAKE_OSX_DEPLOYMENT_TARGET@</string>
+
+		<key>CFBundleDocumentTypes</key>
```

Comment:
should we also add the csv thing here?
https://github.com/jasp-stats/jasp-issues/issues/857

Reply:
I'm not 100% sure, because what we have already doesn't work, so I think we haven't set it up quite right. I need to read about and see what's going on before adding a generic file format to the list.